awx/ui_next/src/util/useWebsocket.js

```diff
@@ -33,9 +33,11 @@ export default function useWebsocket(subscribeGroups) {
     ws.current.onclose = e => {
       // eslint-disable-next-line no-console
       console.debug('Socket closed. Reconnecting...', e);
```

Comment:
We should probably put this log statement inside the `if`, since it doesn't try to reconnect otherwise